### PR TITLE
added info for content pages and added ruby version file AB-7369

### PR DESCRIPTION
**Pull request by andreorvalho**
- This changes the API docs 
 - adds a ruby version file

### CODE REVIEW

**Comment by andreorvalho** [comment]
This is still in progress, but there is no in progress label for this repository...

File: source/includes/_autosignatures.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+```json
+{
+  "autosignatures": [
+    {
+      "autosignature":
+        {
+          "id": 1,
+          "title": "Hello",
+          "signature":"World"
+        }
```
Comment (dipth) [inline source/includes/_autosignatures.md]: You're missing a `,` here

File: source/includes/_mechanic_sub_contractors.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "mechanicSubContractors": [
+    {
+      "id": 1,
+      "chainId": 1,
+      "name": "sub contractor name",
```
Comment (dipth) [inline source/includes/_mechanic_sub_contractors.md]: Please provide some "real" fake values instead

File: source/includes/_mechanic_sub_contractors.md [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+### HTTP Request
+
+`GET https://www.autobutler.dk/api/v2/mechanics/mechanic_sub_contractors`
+
+### Response JSON
+
+Attribute | Type    | Can be blank? | Description
+--------- | ------- | ------------- | --------------------------------------------------------
+id        | integer | no            | The id of the Mechanic sub contractor
+chainId   | integer | no            | The chains Id
```
Comment (dipth) [inline source/includes/_mechanic_sub_contractors.md]: Please provide a more meaningful description of what a chain id is

File: source/includes/_mechanic_sub_contractors.md [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+jQuery.ajax({
+    url: "https://www.autobutler.dk/api/v2/mechanics/mechanic_sub_contractors/1",
+    type: "PATCH",
+    headers: {
+        "Authorization": "token",
+        "Content-Type": "application/json; charset=utf-8",
+    },
+    contentType: "application/json",
+    data: JSON.stringify({
+        "title": "Foo Bar",
```
Comment (dipth) [inline source/includes/_mechanic_sub_contractors.md]: These seem to be copy pasted from another endpoint example without having been updated for this endpoint?

File: source/includes/_autosignatures.md [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+    url: "https://www.autobutler.dk/api/v2/mechanics/autosignatures/1",
+    type: "PATCH",
+    headers: {
+        "Authorization": "token",
+        "Content-Type": "application/json; charset=utf-8",
+    },
+    contentType: "application/json",
+    data: JSON.stringify({
+        "title": "Foo Bar",
+        "text": "My wonderful text"
```
Comment (dipth) [inline source/includes/_autosignatures.md]: These seem to be copy pasted from another endpoint example without having been updated for this endpoint?

File: source/includes/_mechanic_sub_contractors.md [hunk 6]
```diff
@@ -0,0 +1,338 @@
...
+jQuery.ajax({
+    url: "https://www.autobutler.dk/api/v2/mechanics/mechanic_sub_contractors",
+    type: "POST",
+    headers: {
+        "Authorization": "token",
+        "Content-Type": "application/json; charset=utf-8",
+    },
+    contentType: "application/json",
+    data: JSON.stringify({
+        "title": "Wubba Lubba",
```
Comment (dipth) [inline source/includes/_mechanic_sub_contractors.md]: These seem to be copy pasted from another endpoint example without having been updated for this endpoint?

File: source/includes/_content_pages.md [hunk 7]
```diff
@@ -46,7 +46,8 @@ jQuery.ajax({
 }
 ```
 
-This endpoint return a list of all Content Pages available for the workshop.
+This endpoint returns a list of all Content Pages available for
+the workshop ordered by their title.
```
Comment (dipth) [inline source/includes/_content_pages.md]: Please avoid documenting ordering, as the client should never expect single-paged data to be ordered in any specific way. It should instead enforce it's own ordering client-side

File: source/includes/_autosignatures.md (hunk 1 above)
Comment (andreorvalho) [inline source/includes/_autosignatures.md]: yup will fix. Thanks

**Pull request merged**